data/test/scenarios/unknown_scenario_warning.cfg

```diff
@@ -1,6 +1,14 @@
 # wmllint: no translatables
 
-# Check that the "this will lead to an unknown scenario" warning doesn't get triggered.
+#####
+# API(s) being tested: [endlevel]next_scenario=
+##
+# Expected end state:
+# The endlevel for empty_test doesn't print a warning (it exists as another unit test scenario).
+# The endlevel for $chosen_branch doesn't print a warning (it not existing is accepted).
+# The endlevel for victory doesn't print a warning.
+# The dummy containing a next_scenario is ignored.
```

Comment:
The wording feels wrong here, not because the wording itself is wrong but because these tests are inconsistent with the general unit tests. If I was reading this along with the docs of other tests, I would assume this was executing the `[endlevel]` tags.

Might be clearer if it was `API(s) being tested: [scenario]`, because in the nearest ActionWML analogy to what's being tested here, it's checking an action while "executing" the `[test]` tag.